tests/unit/BaseIndexTest.php

```diff
@@ -152,14 +152,35 @@ public function testFacetFields()
         /** @var ArrayList $parents */
         $query->addFacetFilter('Parent', $id);
         $result = $index->buildSolrQuery($query);
-        $filterQuery = $result->getFilterQuery('facets');
+        $filterQuery = $result->getFilterQuery('andFacets');
         $this->assertEquals('SiteTree_ParentID:' . $id, $filterQuery->getQuery());
         $query->addFacetFilter('Parent', 5);
-        $filterQuery = $index->buildSolrQuery($query)->getFilterQuery('facets');
+        $filterQuery = $index->buildSolrQuery($query)->getFilterQuery('andFacets');
         $this->assertContains('SiteTree_ParentID:5 AND SiteTree_ParentID:' . $id, $filterQuery->getQuery());
+        $index->addFacetField(Page::class, ['BaseClass' => SiteTree::class, 'Field' => 'ParentID', 'Title' => 'Child']);
+        $query->addOrFacetFilter('Parent', 3);
+        $query->addOrFacetFilter('Child', 5);
+        $filterQuery = $index->buildSolrQuery($query)->getFilterQuery('orFacet-0');
+        $this->assertNotContains('SiteTree_ParentID:3 AND SiteTree_ParentID:5', $filterQuery->getQuery());
```

Comment:
Trying to understand more :)
```suggestion
        $this->assertNotContains('SiteTree_ParentID:3 AND SiteTree_ParentID:5', $filterQuery->getQuery());
        $this->assertEquals('SiteTree_ParentID:3', $filterQuery->getQuery());
        $filterQuery = $index->buildSolrQuery($query)->getFilterQuery('orFacet-1');
        $this->assertEquals('SiteTree_ParentID:5', $filterQuery->getQuery());
```

Reply:
Ow yes, that's a good addition!